### DISCUSSION TITLE
Rollout of INSPIRE Registry using Re3gistry v.2

### DISCUSSION THREAD

**Discussion by MarcoMinghini** [INSPIRE-MIF/helpdesk-validator#815] [Announcements]
Dear INSPIRE validation community,

this message is to inform you that tomorrow 14 July 2022, the rollout of the [INSPIRE Registry](https://inspire.ec.europa.eu/registry) based on version 2 of the [Re3gistry](https://github.com/ec-jrc/re3gistry/) open source software will happen at 11:00 am CET.

In light of this, a potential service interruption might occur and this can impact the results of the tests performed in the INSPIRE Reference Validator. Therefore, we invite you to wait for some time before reporting issues in the helpdesk.

Kind regards.

**Comment by MarcoMinghini** [comment]
Dear all,
the rollout of the INSPIRE Registry has been postponed. We will keep you informed here about the new target date/time.
Kind regards.

**Comment by juanpelegrina** [comment]
Dear  INSPIRE validation community,
The new rollout of the INSPIRE Registry based on Re3gistry 2.3.1 is re-scheduled on Thursday 28th at 10:00 CEST.
Please beware of potential service interruptions that can impact the results of the tests performed in the INSPIRE Reference Validator at about 10.00.
Kind Regards

**Comment by jenriquesoriano** [comment]
Dear INSPIRE validator community,

we are experiencing some issues with regard to the rollout of the INSPIRE Registry related to the access to Registry resources.
Please excuse the inconvenience. 
We will keep you posted.

Best regards,

**Comment by juanpelegrina** [comment]
Dear INSPIRE validator community,

After the second attempt to rollout the INSPIRE Registry based on Re3gistry 2.3.1 today, we noticed that this affected some Executable Test Suites of the INSPIRE Reference Validator.
As a result, the rollout is postponed again, waiting for preparing proper integration tests between both applications.
We will provide more details about the new rollout date by using this thread (probably to take place in September 2022).

Best regards
